.github/docker-node/node-agent-runner.Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:20.04
+SHELL ["/bin/bash", "-c"]
+
+# general tools
+RUN apt update \
+    && apt -y install \
+                curl \
+                git \
+                g++ \
+                python \
+                make
```

Comment:
Minor: are g++, python and make still needed?  I'm going on the assumption that they were included in order to compile the bindings, not sure if the workflows for agent test/publish will actually have that "fallback to compile" scenario.

Reply:
These are indeed the `node-gyp` dependencies, but they are not included for `appoptics-apm` (though there exist scenarios/setups where building from source may be desired)... they needed for `libpq` (which comes from `pg-native`) and `dtrace-provider` (which comes from `bunyan` and `restify`)...